core/types/istanbul.go

```diff
@@ -88,6 +88,7 @@ type WBFTExtra struct {
 	PreparedSeal      *WBFTAggregatedSeal
 	CommittedSeal     *WBFTAggregatedSeal
 	EpochInfo         *EpochInfo // epoch info is filled only for last block of epoch
+	GasTip            *big.Int   // tip value agreed through governance voting (in Wei)
```

Comment:
```suggestion
	GasTip            *big.Int   // tip value agreed through governance voting (in Wei)
	EpochInfo         *EpochInfo // epoch info is filled only for last block of epoch
```
I think it's better that an epoch block and an non-epoch block have the same structure of block.

Reply:
I applied it.